tests/00_00_04-change-federation.js

```diff
@@ -236,6 +251,111 @@ describe('Change federation', async function() {
 
     });
 
+    it('should create the proposed federation', async () => {
+
+        // Act
+
+        const proposedFederationInfo = await getProposedFederationInfo(bridge);
```

Comment:
I think this is not related to the act of creating the proposed federation, therefore I would put this in the assert section.

Reply:
Updated.